[2024-12-19T05:40:10.027+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-19T05:40:10.036+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag.first_task scheduled__2022-05-13T00:00:00+00:00 [queued]>
[2024-12-19T05:40:10.040+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag.first_task scheduled__2022-05-13T00:00:00+00:00 [queued]>
[2024-12-19T05:40:10.041+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2024-12-19T05:40:10.061+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): first_task> on 2022-05-13 00:00:00+00:00
[2024-12-19T05:40:10.066+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1414) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-19T05:40:10.067+0000] {standard_task_runner.py:72} INFO - Started process 1421 to run task
[2024-12-19T05:40:10.067+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag', 'first_task', 'scheduled__2022-05-13T00:00:00+00:00', '--job-id', '292', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpjyudxh09']
[2024-12-19T05:40:10.068+0000] {standard_task_runner.py:105} INFO - Job 292: Subtask first_task
[2024-12-19T05:40:10.100+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag.first_task scheduled__2022-05-13T00:00:00+00:00 [running]> on host 13c961a72918
[2024-12-19T05:40:10.150+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='code2j' AIRFLOW_CTX_DAG_ID='our_first_dag' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2022-05-13T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-05-13T00:00:00+00:00'
[2024-12-19T05:40:10.151+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-19T05:40:10.165+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2024-12-19T05:40:10.165+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world, this is the first task']
[2024-12-19T05:40:10.172+0000] {subprocess.py:99} INFO - Output:
[2024-12-19T05:40:10.173+0000] {subprocess.py:106} INFO - hello world, this is the first task
[2024-12-19T05:40:10.173+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2024-12-19T05:40:10.191+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-19T05:40:10.192+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag, task_id=first_task, run_id=scheduled__2022-05-13T00:00:00+00:00, execution_date=20220513T000000, start_date=20241219T054010, end_date=20241219T054010
[2024-12-19T05:40:10.207+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-19T05:40:10.218+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-19T05:40:10.219+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
